src/test/java/net/snowflake/client/core/arrow/BigIntToTimeConverterTest.java

```diff
@@ -11,11 +11,7 @@
 import static org.junit.Assert.assertTrue;
 
 import java.sql.Time;
-import java.util.HashMap;
-import java.util.HashSet;
-import java.util.Map;
-import java.util.Random;
-import java.util.Set;
+import java.util.*;
```

Comment:
let's not use wildecard imports - here and in the other places

Reply:
Fixed